cmd/keytransparency-monitor/main.go

```diff
@@ -85,10 +84,8 @@ func main() {
 		glog.Exitf("Failed to initialize monitor: %v", err)
 	}
 
-	// TODO(gbelvin): persist trusted roots
```

Comment:
Is this TODOne? If the monitor restarts, it will start over from rev 0.

Reply:
This is now refactored into the client which will be refactored into a log tracker soon